lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -201,6 +205,10 @@ def _check_postgres_adapter!
     raise PgHaMigrations::UnsupportedAdapter, "This gem only works with the #{expected_adapter} adapter, found #{actual_adapter} instead" unless actual_adapter == expected_adapter
   end
 
+  def _type_is_enum(type)
```

Comment:
Should this also be used in https://github.com/braintree/pg_ha_migrations/pull/43/files#diff-7031f9f7b6a70dc5abef1a41a89aded1R81 ?

Reply:
You already fixed that in https://github.com/jcoleman/pg_ha_migrations/commit/1395fb4178e8fd474028b22c7a81a129125119b6

`column.sql_type_metadata.type` will return `:enum` for an existing column. The problem here is that there isn't a way (or at least I couldn't find a way) to tell if the type being specified for a new column is an enum without making an extra query.

Reply:
Heh. Yeah, we do the same thing in the byte packing code to determine if an enum exists.